.storybook/config.js

```diff
@@ -18,4 +19,6 @@ addParameters({
 addDecorator(withInfo) // info addon must sit on top of other decorators to work properly
 addDecorator(withA11y)
 
+addParameters(keyholeViewports)
```

Comment:
Can you put the addParameters together or is there a reason why this is here?

Reply:
AH. for some reason i assumed that the addParameters works the same way as addDecorator and must be split out 😅